docker/image_types.yaml

```diff
@@ -3,17 +3,19 @@
 
 'base-image':
   tag: 'base-image'
-  context: 'docker/base-image'
+  dockerfile: 'docker/base-image/Dockerfile'
```

Comment:
Note that it's not necessary to explicitly specify the dockerfile, when there's a Dockefile in the context root (e.g. in this case)

Reply:
Right and I asked for this change so we don't have to special case images without dockerfiles during generation. The extra line isn't a problem IMO and is clearer for someone who doesn't know this.